src/views/Dashboard.js

```diff
@@ -148,7 +148,7 @@ const Dashboard = () => {
             <ButtonDouble
               leftBtnPath={SEARCH_PATH}
               leftBtnText={translate('Dashboard.buttonTrustPeople')}
-              rightBtnPath={SEND_PATH}
+              rightBtnPath={SHARE_PATH}
```

Comment:
this is incorrect
Dashboards should have trust and send. 
MyProfile edit and share